py-polars/polars/io/csv/functions.py

```diff
@@ -133,7 +133,8 @@ def read_csv(
         ``utf8-lossy``, the input is first decoded in memory with
         python. Defaults to ``utf8``.
     low_memory
-        Reduce memory usage at expense of performance.
+        Reduce memory usage at expense of performance when rechunking into
+        a single array. To work with larger than-memory datasets use streaming mode.
```

Comment:
It's still not clear to me what this parameter does exactly. The note about streaming shouldn't be in the description text but you could add it in a note adminition, e.g. `.. note::`

Reply:
Some more text added and comment on streaming moved to a note